forte/utils/utils.py

```diff
@@ -127,6 +130,57 @@ def get_class(full_class_name: str, module_paths: Optional[List[str]] = None):
     return class_
 
 
+@lru_cache()
+def cached_locate(name_to_locate_class):
+    return locate(name_to_locate_class)
+
+
+def get_class(full_class_name: str, module_paths: Optional[List[str]] = None):
+    r"""Returns the class based on class name, with cache to improve speed.
```

Comment:
explain what is being cached (and maybe how the cache work)

Reply:
This is now fixed: refactored code to combine the 2 methods into one ("get_class") with "cached_lookup" optional parameter.